atomate/vasp/workflows/base/library/neb.yaml

```diff
@@ -0,0 +1,53 @@
+# A standard CI-NEB calculation
```

Comment:
There are several options to set up image numbers:
  1. Use yaml file: set "IMAGES" tag for NEB firework user_inar_settings (**wf_nudged_elastic_band** method in atomate.vasp.workflow.presets.core)
  2. Or it is the same to set user_incar_settings for every step in **get_wf_neb_from_structure**, **get_wf_neb_from_endpoints** or **get_wf_neb_from_images** (atomate.vasp.workflow.base.neb)
  3. If above settings are not given, image number will be determined by ```IMAGES = endpoint_distance / image_distance```. Default is ```image_dist = 0.7``` angstrom.

After this, the node number to run workflow will be updated from 1 to image number.